src/tooltip/tooltip.js

```diff
@@ -442,7 +470,11 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
               if (triggers.show !== 'none') {
                 triggers.show.forEach(function(trigger, idx) {
                   // Using raw addEventListener due to jqLite/jQuery bug - #4060
-                  if (trigger === triggers.hide[idx]) {
+                  if (trigger === 'outsideClick') {
+                    element[0].addEventListener('click', toggleTooltipBind);
+                    $document[0].body.addEventListener('click', bodyHideTooltipBind);
```

Comment:
I think it's better to have the listener on $document, for consistency with the rest of the library.